lenses/httpd.aug

```diff
@@ -59,7 +59,7 @@ let empty               = Util.empty_dos
 let indent              = Util.indent
 
 (* borrowed from shellvars.aug *)
-let char_arg_dir  = /[^\\ '"\t\r\n]|\\\\"|\\\\'/
+let char_arg_dir  = /([^\\ '"\t\r\n]|[^\\ '"\t\r\n][^ '"\t\r\n]*[^\\ '"\t\r\n])|\\\\"|\\\\'/
```

Comment:
I can't review this for correctness yet but I'm going to try to describe what this changes in English.  

Previously, change_arg_dir could be any single non(backslash or "terminator") , or an escaped single or double quote.

Now change_arg_dir could be those things, or any string of two or more non"terminator" chacters, with backslashes permitted in any position except the first and last in the string.

"terminator" means whitespace or quote.